src/vm/jitinterface.cpp

```diff
@@ -10882,6 +10882,31 @@ void CEEJitInfo::CompressDebugInfo()
     EE_TO_JIT_TRANSITION();
 }
 
+void increaseUnwindInfoSize(ULONG &unwindSize)
+{
+#if defined(_TARGET_AMD64_) || defined(_TARGET_X86_)
+    // Add space for personality routine, it must be 4-byte aligned.
+    // Everything in the UNWIND_INFO up to the variable-sized UnwindCodes
+    // array has already had its size included in unwindSize by the caller.
+    unwindSize += sizeof(ULONG);
```

Comment:
We should not need the personality routine on x86. Also, the x64 unwind info codes referred to in next comment do not make sense on x86.